_posts/en/newsletters/2019-11-13-newsletter.md

```diff
@@ -0,0 +1,367 @@
+    incorporate by reference an established language for
+    machine-readable invoices, such as the [Universal Business
+    Language][] (UBL).  However, there was a concern that implementing
+    the full UBL specification would be an excessive burden on developers
+    of LN software.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #16110][] adds support for building Bitcoin Core (including
+  the GUI) for the Android Native Development Kit (NDK).  In contrast to
+  independent projects like [Android Bitcoin Core][] (ABcore) that build
+  their own Bitcoin Core binaries for Android NDK, adding support
+  directly to the Bitcoin Core project may simplify testing and could
+  allow future PRs to add the Android builds to the set of
+  deterministically-generated executables that allow users to receive
+  greater assurances that they are running the same well-reviewed code
+  that exists in the code repository.
+
+- [Bitcoin Core #16899][] adds a `dumptxoutset` RPC that will write a
+  copy of the current UTXO set to disk in a serialization format
+  designed for future use with nodes bootstrapped using assumeutxo (see
+  [Newsletter #41][news41 assumeutxo]).  Additionally, a script is added to the
+  project's contributor tools that will rewind the block chain to a
+  specified block height, dump the UTXO set at that point, and then
+  start reprocessing blocks normally again.  This can take several
+  seconds per block, so running this script for a height thousands of
+  blocks in the past can take a very long time.
+
+- [Bitcoin Core #17258][] updates the `listsinceblock` RPC to prevent
+  listing any transactions that can't be confirmed because another
+  transaction spent at least one of the same inputs and was already
+  included in the block chain prior to the blocks being evaluated by the
+  RPC call.
+
+- [C-Lightning #3246][] attempts to fix the potential data leak
+  described on the LN mailing list this week (see the [news item][news72
+  sphinx]).
+
+- [LND #3442][] makes it possible for a spender to manually construct
+  packets for simple multipath payments---payments that are split into
+  parts and which can be independently routed through different
+  channels.  This isn't meant to be accessed by users but rather to be
+  built upon by subsequent features that add other functions related to
+  multipath payments.  For more about multipath payments, see
+  [Newsletter #27][news27 multipath].
+
+- [BIPs #857][] edits [BIP157][] to increase the maximum number of
+  compact block filters that can be requested at once from 100 to 1,000.
+  This reverts a [PR from last year][BIPs #699] that lowered it from
+  1,000 to 100.
```

Comment:
It seems this entry is missing the usual explanation of why this was done, e.g. to improve node' sync speed via better batching requests for filters, IIUC

Reply:
I do like to include explanations in merge descriptions, but the justification for this as offered in the PR doesn't make sense to me.  It talks about round trips, but I don't know of any reason why you have to wait for the response to one `getcfilters` request in order to send a second request.  I wonder if it was just easier to change the BIP than it was to change all the software using the 1,000-filter value.

In any case, I don't think this is an important-enough change to really need a justification.  It's just an arbitrary number change.